doc/radosgw/multisite.rst

```diff
@@ -6,33 +6,39 @@ Multi-Site
 
 .. versionadded:: Jewel
 
-A single zone configuration typically consists of one zone group containing one
-zone and one or more `ceph-radosgw` instances where you may load-balance gateway
-client requests between the instances. In a single zone configuration, typically
-multiple gateway instances point to a single Ceph storage cluster. However, Kraken
-supports several multi-site configuration options for the Ceph Object Gateway:
-
-- **Multi-zone:** A more advanced configuration consists of one zone group and
-  multiple zones, each zone with one or more `ceph-radosgw` instances. Each zone
-  is backed by its own Ceph Storage Cluster. Multiple zones in a zone group
-  provides disaster recovery for the zone group should one of the zones experience
-  a significant failure. In Kraken, each zone is active and may receive write
-  operations. In addition to disaster recovery, multiple active zones may also
-  serve as a foundation for content delivery networks.
-
-- **Multi-zone-group:** Formerly called 'regions', Ceph Object Gateway can also
-  support multiple zone groups, each zone group with one or more zones. Objects
-  stored to zones in one zone group within the same realm as another zone
-  group will share a global object namespace, ensuring unique object IDs across
-  zone groups and zones.
-
-- **Multiple Realms:** In Kraken, the Ceph Object Gateway supports the notion
-  of realms, which can be a single zone group or multiple zone groups and
-  a globally unique namespace for the realm. Multiple realms provide the ability
-  to support numerous configurations and namespaces.
-
-Replicating object data between zones within a zone group looks something
-like this:
+A single-zone configuration typically consists of (1) one "zone group", which
+contains one zone and (2) one or more `ceph-radosgw` instances between which
```

Comment:
s/between/among?

Reply:
I feel the weighing of these two prepositions in the semantic scales of my mind, but my scales tip the other way here. I'll bear this in mind over the next few days as I revise this file.

Also, I should probably say that all of these edits are being made so that I can get straight in my head what this RGW page really means to say so that I can answer Alexander Patrakov's request for a conceptual overview of RGW at the top of this page. The idea is to give the reader a kind of aerial view of RGW so that it is easier to navigate in the field.  All of that is a roundabout way of saying that your counsel will be thirstily sought in matters such as the matter of this evaluation of prepositions.

I'm going to let this stand for the moment as it is. But I might suddenly be persuaded for God knows what reason, and then change this text from "between" to "among".

Reply:
You're the boss.  FWIW I was taught that with three or more objects of a preposition, "among" should be used, "between" only for two.

Reply:
Notes for a better response:

cf. Garner's Modern English Usage (2016), p.111